xfel/command_line/xtc_dump.py

```diff
@@ -141,11 +143,13 @@ def run(self):
       else:
         tmp_dir = os.path.join(params.output.tmp_output_dir, '.tmp')
       if not os.path.exists(tmp_dir):
-        try:
-          os.makedirs(tmp_dir)
-        except Exception as e:
-          if not os.path.exists(tmp_dir):
-            halraiser(e)
+        with show_mail_on_error():
+          try:
+            os.makedirs(tmp_dir)
+            # Can fail if running multiprocessed - that's OK if the folder was created
+          except OSError as e:  # In Python 2, a FileExistsError is just an OSError
+            if e.errno != errno.EEXIST:  # If this OSError is not a FileExistsError
+              raise
```

Comment:
OK, I thought GitHub made it easier to see the location of a commit comment in a PR.  Here's the first location...